util/Setup/Configuration.cs

```diff
@@ -80,7 +80,7 @@ public class Configuration
             "child-src 'self' https://*.duosecurity.com https://*.duofederal.com; " +
             "frame-src 'self' https://*.duosecurity.com https://*.duofederal.com; " +
             "connect-src 'self' wss://{0} https://api.pwnedpasswords.com " +
-            "https://twofactorauth.org; object-src 'self' blob:;";
+            "https://twofactorauth.org https://2fa.directory; object-src 'self' blob:;";
```

Comment:
We can probably remove the old url since we don't expect it to come back.

Reply:
I had left the old URL in prod to handle legacy clients, pre-stage, but yeah, I agree